lib/msal-react-native-android-poc/README.md

```diff
@@ -0,0 +1,98 @@
+
+For a complete listing of each method's return value and map keys, see the Methods(link to Methods) section.
+
+## What to expect from this library
+Because this library is a proof of concept, it is scoped to specific features and does not implement all the capabilities of an official MSAL for React Native library. 
+
+This proof of concept is scoped to consider:
+* Compatibility with the Android platform
+* Authenticating users with work and school Azure AD accounts (AAD)
+* Single-account mode
+
+This proof of concept does not implement:
+* Compatibility with iOS
+    * To allow compatibility with iOS, an [Objective-C native module](https://reactnative.dev/docs/native-modules-ios) must be created that wraps the       [MSAL for iOS and macOS](https://github.com/AzureAD/microsoft-authentication-library-for-objc#:~:text=The%20MSAL%20library%20for%20iOS%20and%20macOS%20gives,for%20those%20using%20our%20hosted%20identity%20management%20service.).
+* B2C or AD FS
+    * For implementing B2C, refer to the Microsoft docs on using [MSAL for Android with B2C](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-android-b2c). Since the returned value from acquiring tokens is an [`IAuthenticationResult`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.identity.client.authenticationresult?view=azure-dotnet), the results of the wrapped B2C methods can be handled in the same manner as the existing AAD methods.
+* Multiple-account mode
+    * This could be as simple as adding an instance of [MultipleAccountPublicClientApplication](https://docs.microsoft.com/en-us/java/api/com.microsoft.identity.client.multipleaccountpublicclientapplication?view=azure-java-stable) to the existing native module file and      wrapping its corresponding methods.
+    
+Additionally, with this library being a proof of concept, it is not recommended to be used for React Native applications intended for production.
+
+## Methods
+This library's methods wrap MSAL for Android's corresponding `singleAccountPublicClientApplication` methods. All methods return a promise that, in most cases, resolves to a map and returns an exception if rejected. Please see the Microsoft Docs for [`singleAccountPublicClientApplication`](https://docs.microsoft.com/en-us/java/api/com.microsoft.identity.client.singleaccountpublicclientapplication?view=azure-java-stable) for more general information regarding MSAL methods.
+Method | Description
+------------ | -------------
+signIn(String scopesValue) | Directs a user to sign in with an AAD account. scopesValue is a String containing scopes separated by spaces (" "). Returns a map with the `authority`, `id`, `tenantId`, `username`, and `idToken` of the account. See the Microsoft docs on [IAccount](https://docs.microsoft.com/en-us/java/api/com.microsoft.identity.client.iaccount?view=azure-java-stable) for more details of the map's keys.
```

Comment:
Not the biggest deal, but this table could definitely be cleaned up for readability in the markdown

Reply:
added some spacing to hopefully improve readability.